### DISCUSSION TITLE
Grafana Post-install

### DISCUSSION THREAD

**Discussion by eteixido1** [tteck/Proxmox#1464] [Help Needed]
Hello tteck,

Somthing goes wrong with grafana post-install. The script seems to finish without errors but /etc/grafana/grafana.ini are comented. I tryed with alpine and debian and the problem is the same.

I uncomment port 3000 in the config file and restart the service and then Grafana starts without any problem. I don't know if I need to configure somthing more.

Kind regards,

**Comment by tteck** [comment]
I can't replicate 

```   ______           ____                 
  / ____/________ _/ __/___ _____  ____ _
 / / __/ ___/ __  / /_/ __  / __ \/ __  /
/ /_/ / /  / /_/ / __/ /_/ / / / / /_/ / 
\____/_/   \__,_/_/  \__,_/_/ /_/\__,_/  
 
Using Advanced Settings
Using Distribution: debian
Using debian Version: 11
Using Container Type: 1
Using Root Password: Automatic Login
Container ID: 112
Using Hostname: grafana
Using Disk Size: 2
Allocated Cores: 1
Allocated RAM: 512
Using Bridge: vmbr0
Using IP Address: dhcp
Using Gateway IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using MAC Address: AE:1A:60:07:B7:59
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: no
Creating a Grafana LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 112 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.253 
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.112.4
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Set up Grafana Repository
 ✓ Installed Grafana
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!

Grafana should be reachable by going to the following URL.
         http://192.168.86.253:3000 
```
```
#################################### Server ####################################
[server]
# Protocol (http, https, h2, socket)
;protocol = http

# The ip address to bind to, empty will bind to all interfaces
;http_addr =

# The http port  to use
;http_port = 3000

# The public facing domain name used to access grafana from a browser
;domain = localhost

# Redirect to correct domain if host header does not match domain
# Prevents DNS rebinding attacks
;enforce_domain = false

# The full public facing url you use in browser, used for redirects and emails
# If you use reverse proxy and sub path specify full url (with sub path)
;root_url = %(protocol)s://%(domain)s:%(http_port)s/

```
![Screenshot 2023-06-07 11 13 36 AM](https://github.com/tteck/Proxmox/assets/21342670/f37241b8-11ec-4ba1-8bb2-3ac1dec5d736)

**Reply by eteixido1** [reply]
> I can't replicate
> 
> ```
>   / ____/________ _/ __/___ _____  ____ _
>  / / __/ ___/ __  / /_/ __  / __ \/ __  /
> / /_/ / /  / /_/ / __/ /_/ / / / / /_/ / 
> \____/_/   \__,_/_/  \__,_/_/ /_/\__,_/  
>  
> Using Advanced Settings
> Using Distribution: debian
> Using debian Version: 11
> Using Container Type: 1
> Using Root Password: Automatic Login
> Container ID: 112
> Using Hostname: grafana
> Using Disk Size: 2
> Allocated Cores: 1
> Allocated RAM: 512
> Using Bridge: vmbr0
> Using IP Address: dhcp
> Using Gateway IP Address: Default
> Disable IPv6: yes
> Using Interface MTU Size: Default
> Using DNS Search Domain: Host
> Using DNS Server IP Address: Host
> Using MAC Address: AE:1A:60:07:B7:59
> Using Vlan: Default
> Enable Root SSH Access: no
> Enable Verbose Mode: no
> Creating a Grafana LXC using the above advanced settings
>  ✓ Using local for Template Storage.
>  ✓ Using local-lvm for Container Storage.
>  ✓ Updated LXC Template List
>  ✓ LXC Container 112 was successfully created.
>  ✓ Started LXC Container
>  ✓ Set up Container OS
>  ✓ Network Connected: 192.168.86.253 
>  ✓ Internet Connected
>  ✓ DNS Resolved github.com to 140.82.112.4
>  ✓ Updated Container OS
>  ✓ Installed Dependencies
>  ✓ Set up Grafana Repository
>  ✓ Installed Grafana
>  ✓ Customized Container
>  ✓ Cleaned
>  ✓ Completed Successfully!
> 
> Grafana should be reachable by going to the following URL.
>          http://192.168.86.253:3000 
> ```
> 
> ```
> #################################### Server ####################################
> [server]
> # Protocol (http, https, h2, socket)
> ;protocol = http
> 
> # The ip address to bind to, empty will bind to all interfaces
> ;http_addr =
> 
> # The http port  to use
> ;http_port = 3000
> 
> # The public facing domain name used to access grafana from a browser
> ;domain = localhost
> 
> # Redirect to correct domain if host header does not match domain
> # Prevents DNS rebinding attacks
> ;enforce_domain = false
> 
> # The full public facing url you use in browser, used for redirects and emails
> # If you use reverse proxy and sub path specify full url (with sub path)
> ;root_url = %(protocol)s://%(domain)s:%(http_port)s/
> ```
> 
> ![Screenshot 2023-06-07 11 13 36 AM](https://user-images.githubusercontent.com/21342670/244114043-f37241b8-11ec-4ba1-8bb2-3ac1dec5d736.png)

Hello tteck,

With debian and alpine I've need to remove ; before the port and restart the service. Without that the url is unreachable.

Kind regards,
